docs/management/admin/endpoint-command-ref.asciidoc

```diff
@@ -0,0 +1,343 @@
+[[endpoint-command-ref]]
+= {elastic-endpoint} command reference
+
+This page lists the commands for management and troubleshooting of {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention.
+
+[NOTE]
+====
+* The service is not added to the `PATH` system variable, so you must prepend the commands with the full OS-dependent path:
+** On Windows: `"C:\Program Files\Elastic\Endpoint\elastic-endpoint.exe"`
+** On macOS: `/Library/Elastic/Endpoint/elastic-endpoint`
+** On Linux: `/opt/Elastic/Endpoint/elastic-endpoint`
+
+* You must run the commands with elevated privileges—as the root user on POSIX systems, or as Administrator on Windows.
+====
+
+The following {elastic-endpoint} commands are available:
+
+* <<elastic-endpoint-diagnostics-command, diagnostics>>
+* <<elastic-endpoint-help-command, help>>
+* <<elastic-endpoint-inspect-command, inspect>>
+* <<elastic-endpoint-install-command, install>>
+* <<elastic-endpoint-memorydump-command, memorydump>>
+* <<elastic-endpoint-run-command, run>>
+* <<elastic-endpoint-send-command, send>>
+* <<elastic-endpoint-status-command, status>>
+* <<elastic-endpoint-test-command, test>>
+* <<elastic-endpoint-top-command, top>>
+* <<elastic-endpoint-uninstall-command, uninstall>>
+* <<elastic-endpoint-version-command, version>>
+
+Each of the commands accepts the following logging options:
+
+* `--log [stdout,stderr,debugview,file]`
+* `--log-level [error,info,debug]`
+
+[discrete]
+[[elastic-endpoint-diagnostics-command]]
+== elastic-endpoint diagnostics
+
+Gather diagnostics information from {elastic-endpoint}. This command produces an archive that contains:
+
+- `version.txt`: Version information
+- `elastic-endpoint.yaml`: Current policy
+- `metrics.json`: Metrics document
+- `policy_response.json`: Last policy response
+- `system_info.txt`: System information
+- `analysis.txt`: Diagnostic analysis report
+- `logs` directory: Copy of {elastic-endpoint} log files
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint diagnostics
+------
+
+[discrete]
+[[elastic-endpoint-help-command]]
+== elastic-endpoint help
+
+Show help for the available commands.
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint help
+------
+
+[discrete]
+[[elastic-endpoint-inspect-command]]
+== elastic-endpoint inspect
+
+Show the current {elastic-endpoint} configuration.
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint inspect
+------
+
+[discrete]
+[[elastic-endpoint-install-command]]
+== elastic-endpoint install
+
+Install {elastic-endpoint} as a system service.
+
+NOTE: Elastic doesn't publish independent {elastic-endpoint} packages since {elastic-endpoint} is managed by {agent}.
```

Comment:
What exactly should the user conclude from this, and what action do we want them to take with this information? Should they _not_ install Endpoint like this, if it's limited in this way? That's kinda what it sounds like, but it's so indirect it's hard to tell.

Reply:
Hi, I wrote the draft, the point here was that users reading this documentation shouldn't start googling (or asking in the community) where to get the Endpoint installation package from. Indeed it also implies they have no business trying to install it by this command. Endpoint is unusable without Agent.

I wont mind to omit this command from the docs entirely if it makes more sense 😉

Reply:
Thanks for the helpful context @intxgo! I could see us either omitting the command for those reasons, or including it but with an admonition that basically tells the user don't bother using it because standalone Endpoint (without Agent) won't work. 

The main reason I suggest including it is for completeness; if the user can run `elastic-endpoint help` and the output includes `install`, then they might be tempted to try it out.

Reply:
agree, just feel free to reword the note part to make it clearer